develop/processor/index.md

```diff
@@ -14,6 +14,8 @@ public static void main(String[] args) {
 	VeraGreenfieldFoundryProvider.initialise();
 	// Default validator config
 	ValidatorConfig validatorConfig = ValidatorFactory.defaultConfig();
+	// or it is possible to select the needed parameters using ValidatorConfigBuilder, for example flavour
```

Comment:
// or it is possible to specify required parameters using ValidatorConfigBuilder. For example, to specify a validation flavour explicitly: